modules/external-secret-store/providers.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">= 0.13"
+
+  required_providers {
+    kubectl = {
```

Comment:
aws also need to be included? and also will be good to rename this file to versions.tf as we have with new best practice/standard way to create modules?